ajax/libs/sentinel-js/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "sentinel-js",
+  "filename": "sentinel.min.js",
+  "license": "MIT",
+  "description": "JS library that detects new DOM nodes using CSS selectors",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/muicss/sentineljs.git"
+  },
+  "author": {
+    "name": "https://www.muicss.com",
+    "email": "andres@muicss.com",
+    "url": "Andres Morey"
```

Comment:
This should be `https://www.muicss.com`

Reply:
@extend1994 Hi, I just committed the changes. However, the previous commit messages that were deleted by @PeterDaveHello are back again. Is this correct? I rebased first before I made the changes to package.json.